python/feathub/feature_views/feature.py

```diff
@@ -54,6 +55,7 @@ def __init__(
                      the `transform` or the parent table's keys.
         :param input_features: The names of fields in the parent table used by
                               `transform` to derive this feature's values.
+        :param attributes: The extra attribute of the feature that are defined by user.
```

Comment:
nit: change `The extra attribute` to `The extra attributes`. Same for other classes.